lib/ghtorrent/command.rb

```diff
@@ -94,7 +94,7 @@ def process_options
             :type => String
         opt :token, 'GitHub OAuth token',
             :type => String, :short => 't'
-        opt :req_limit, 'Request limit for provided account (in reqs/hour)',
+        opt :req_limit, 'Number or requests to leave on any provided account (in reqs/hour)',
```

Comment:
How do you feel about this re-orientation of req_limit?  The idea is that we want to ensure a certain amount is left,  That eliminates the need to know how many are on any given key (we have a mix of big and normal keys).  It does however change a user input.  We could do a separate option (name?) and have them be mutually exclusive.